great_expectations/checkpoint/actions.py

```diff
@@ -439,32 +450,42 @@ def _run(  # type: ignore[override] # signature does not match parent  # noqa: P
             )
 
         validation_success = validation_result_suite.success
+        expectation_suite_name = validation_result_suite.meta.get(
+            "expectation_suite_name", "__no_expectation_suite_name__"
+        )
+
+        return self._run_pypd_alert(
+            name=expectation_suite_name,
+            template="Great Expectations suite check {name} has failed",
```

Comment:
This is to maintain existing behavior until we rip out single validation result support. Maybe not the cleanest approach but it does let us leverage a lot of the same code for both paths

Reply:
`_run_pypd_alert` is a new method you've added. It seems strange to pass in `name` and then have that used in a `template`. It begs the question, why not pass in the actual message instead of a template. I see `name` is used as the `dedup_key` as well. I'd change the signature of this new method so `name` -> `dedup_key` or just `key` and change template to `summary`.